models/contact.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const { stringValidation } = require('../helpers/validation/modelValidation')
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Contact extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    // static associate(models) {
+    //   // define association here
+    // }
+  };
+  Contact.init({
+      name: {
+        type: DataTypes.STRING(100),
+        allowNull: false,
+        validate: {
+          is: {
+              args: stringValidation(2, 100),
+              msg: "Invalid name",
+          },
+        }
+      },
+      phone: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        validate: {
+          isInt: true,
+        }
+      }, 
+      email: {
+        type: DataTypes.STRING(100),
+        allowNull: false,
+        validate: {
+          isEmail: true,
+          is: {
+            msg: "Invalid email",
+          },
```

Comment:
el "Invalid email" no era un mensaje de error general, sino puntualmente del StringValidation. Ahora te quedo una estructura incompleta para la validación custom que representa el `is:{}`. No se si ira a generar un error